worker-document-archetype/documentation/WorkerArchetypeUsage.md

```diff
@@ -158,6 +161,26 @@ The default Document Worker configuration file checks for values as below;
 | maxBatchSize  |  `CAF_WORKER_MAX_BATCH_SIZE` | 2  |
 | maxBatchTime  |  `CAF_WORKER_MAX_BATCH_TIME` | 5000  |
 
+## Deployment
+
+The Document Worker Archetype will generate a `deployment` folder as part of the new project.
+
+This folder contains the following files:
+- docker-compose-&lt;workername&gt;.yml
+    - This file contains the service which can be added to the `docker-compose.yml` file in [data-processing-service-deploy](https://github.hpe.com/caf/data-processing-service-deploy). Documentation on using the `docker-compose.yml` file can be found at https://pages.github.hpe.com/caf/data-processing-service/pages/en-us/Getting-Started.
+- docker-compose-&lt;workername&gt;-debug.yml
```

Comment:
I want the filename changed from "docker-compose-&lt;workername&gt;-debug.yml" to "docker-compose-&lt;workername&gt;.debug.yml".